### DISCUSSION TITLE
Aggregate events into an Incident

### DISCUSSION THREAD

**Discussion by 0xab3d** [wazuh/wazuh#16430] [Ideas]
Hello,

In my opinion Wazuh is so noisy, it would be great to allocate a place for creating incidents and link them to specific events. Also having the capability to manually link a specific event to an existing incident.

What I mean is to have the capability to do something like USM Alarms and QRadar Offenses.

**Comment by mdiego92** [comment]
Hello @0xab3d,

It is possible to use integrations to external applications/services that may help you with the incident creation related to an alert if that's what you meant. 
Documentation for reference: https://wazuh.com/blog/how-to-integrate-external-software-using-integrator/

Regards!

**Reply by 0xab3d** [reply]
Thanks @mdiego92 

I am already checking out The Hive Project. If you are able to close the issue please feel free. Thanks a lot for your assistance.

**Reply by mdiego92** [reply]
Hey @0xab3d,

No problem. Now that you mention The Hive, please feel free to check our article for [Using Wazuh and TheHive for threat protection and incident response](https://wazuh.com/blog/using-wazuh-and-thehive-for-threat-protection-and-incident-response/).

Thank you very much!

**Reply by 0xab3d** [reply]
Will do. Thank you 😊
